Benchmarking: ModuleNotFoundError: No module named 'pkg_resources'

Currently right after the first benchmark results is printed we get an error `No module named 'pkg_resources'`.
Usually this happens if you update Python `setuptools` to version >= `82.0.0` because starting from version `82.0.0` setuptools no longer includes this module. However, could be any other transitive dependency update.

For confirmed bugs, please report:

- Version: `master`
- Storage Backend: -
- Mixed Index Backend: -
- Link to discussed bug: -
- Expected Behavior: Benchmark data collection should not fail
- Current Behavior: Benchmark data collection fails
- Steps to Reproduce:

See failed CI for steps to reproduce: https://github.com/JanusGraph/janusgraph/actions/runs/28296682635/job/83837942320

### Stack Trace (if you have one)

Right after the first benchmark results are printed we get the following trace:
```
18:07:18 ERROR com.datastax.oss.driver.api.testinfra.ccm.CcmBridge$2.processLine - ccmerr> Traceback (most recent call last):
18:07:18 ERROR com.datastax.oss.driver.api.testinfra.ccm.CcmBridge$2.processLine - ccmerr>   File "/opt/hostedtoolcache/Python/3.14.6/x64/bin/ccm", line 23, in <module>
18:07:18 ERROR com.datastax.oss.driver.api.testinfra.ccm.CcmBridge$2.processLine - ccmerr>     import pkg_resources
18:07:18 ERROR com.datastax.oss.driver.api.testinfra.ccm.CcmBridge$2.processLine - ccmerr> ModuleNotFoundError: No module named 'pkg_resources'
Error: Exception in thread "main" java.lang.RuntimeException: The command '[ccm, create, ccm_1, -i, 127.0.0., -n, 1:0, -v, 4.0.0, --config-dir=/tmp/ccm5727298256338999495]' failed to execute
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.execute(CcmBridge.java:420)
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.execute(CcmBridge.java:373)
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.execute(CcmBridge.java:357)
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.create(CcmBridge.java:229)
	at org.janusgraph.BenchmarkRunner.runCqlBenchmarks(BenchmarkRunner.java:73)
	at org.janusgraph.BenchmarkRunner.main(BenchmarkRunner.java:115)
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:355)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:253)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:236)
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.execute(CcmBridge.java:412)
	... 5 more
Error: Exception in thread "Thread-67" java.lang.RuntimeException: The command '[ccm, remove, --config-dir=/tmp/ccm5727298256338999495]' failed to execute
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.execute(CcmBridge.java:420)
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.execute(CcmBridge.java:373)
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.execute(CcmBridge.java:357)
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.remove(CcmBridge.java:312)
	at org.janusgraph.BenchmarkRunner.lambda$runCqlBenchmarks$0(BenchmarkRunner.java:69)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:355)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:253)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:236)
	at com.datastax.oss.driver.api.testinfra.ccm.CcmBridge.execute(CcmBridge.java:412)
	... 5 more
Error:  Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:355)
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:253)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:884)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:844)
    at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:450)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:919)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:285)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:207)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:362)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
```

--- a/.github/workflows/ci-benchmark.yml
+++ b/.github/workflows/ci-benchmark.yml
@@ -58,7 +58,9 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: '3.x'
-      - run: pip install setuptools
+      # ccm requires the pkg_resources module which was removed in setuptools 82,
+      # so setuptools needs to stay below version 82 (see #4905)
+      - run: pip install 'setuptools<82'
       - run: pip install git+https://github.com/li-boxuan/ccm.git
 
       - uses: actions/checkout@v4

